### PR TITLE
fix: add comment to primeng on table site

### PR DESCRIPTION
**Pull request by cedic3**

### CODE REVIEW

**Comment by gfellerph** [comment]
@cedic3 thanks for submitting a PR! We're having a look at it.

File: packages/documentation/src/stories/components/table/table.docs.mdx [hunk 1]
```diff
@@ -25,3 +25,8 @@ import StylesPackageImport from '../../../shared/styles-package-import.mdx';
 When buttons are placed in a table, be sure to use `alignment: middle`, respectively `<tbody class="align-middle">` to nicely align the content and the buttons.
 
 <Canvas of={TableStories.TableWithButtons} />
+
+### Interactive Table in Angular
+
+If an interactive data table is needed, the design system recommends the table from primeng. However, these can only be used for Angular. The documentation contains all the necessary information on how to use the primeng-table. The LTS versions would need a license, so it is recommended not to use LTS.
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/components/table/table.docs.mdx:31]:
```suggestion
If an interactive data table is needed, we recommend the table component from the PrimeNG UI Suite. To make it look like a Swiss Post component, please use our [@swisspost/design-system-styles-primeng](/?path=/docs/d2112bed-c611-4098-a1ad-e654f7d622e7--docs) package.

However, this component can only be used in Angular projects. Their [documentation](https://primeng.org/table) contains all the necessary information on how to use it.

The LTS versions would need a license, so it's recommended not to use LTS.
```

Comment (oliverschuerch) [reply packages/documentation/src/stories/components/table/table.docs.mdx:31]: @cedic3 The docs page for the PrimeNG table styles package is not yet merged/released, but among the latest PRs #2718

**Review by oliverschuerch — DISMISSED** [review]

**Comment by oliverschuerch** [comment]
Hi @cedic3
Could you please update your branch to the newest `main` branch?
We will try to merge it after the primeng package has been merged and released ;)

- [ ] #2718

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**